src/app/components/sprint/sprint.component.html

```diff
@@ -1,7 +1,6 @@
  <md-list-item *ngIf="sprint">
-  <a md-line href="...">{{ sprint.name }}</a>
+  <a md-line href="#" (click)="onSelect($event, sprint)">{{ sprint.name }}</a>
```

Comment:
Please name click method properly. Name all methods depending on what they do, not when they are called.